src/tracing/librbd.tp

```diff
@@ -773,6 +773,28 @@ TRACEPOINT_EVENT(librbd, invalidate_cache_exit,
     )
 )
 
+TRACEPOINT_EVENT(librbd, discard_image_cache_enter,
```

Comment:
Nit: no need to add new tracepoints. Nothing uses them.

Reply:
Removed